src/main/java/org/broadinstitute/consent/http/db/DataSetDAO.java

```diff
@@ -90,6 +94,9 @@
     @SqlUpdate("update dataset set needs_approval = :needs_approval where dataSetId = :dataSetId")
     void updateDataSetNeedsApproval(@Bind("dataSetId") Integer dataSetId, @Bind("needs_approval") Boolean needs_approval);
 
+    @SqlUpdate("UPDATE dataset SET update_date = :updateDate, update_user_id = :updateUserId WHERE datasetid = :datasetId")
```

Comment:
This only updates the update date and update user id, which could be slightly misleading since I don't change any fields on the dataset itself here, but this indicates when changes occurred in the associated dataset properties (which only store a createDate).

Reply:
Maybe rename this `updateDatasetUpdateUserAndDate` to clarify what it really does.